src/openforms/contrib/referentielijsten/helpers.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime, timedelta
+
+from django.utils import timezone
+
+
+def tabel_or_item_has_expired(einddatum: str) -> bool:
```

Comment:
```suggestion
def check_expired(einddatum: str) -> bool:
```
there's nothing that ties this to a table or item, so we can use a more generic function name. Also, if it returns a boolean, it's nice to use a `is_` prefix or use `check_` in the name to hint about the return type.